model/canvas.cc

```diff
@@ -697,8 +697,6 @@ namespace minsky
 	           
                 selection.autoLayout();
 	           
-                if (!existingParms.empty()) existingParms.clear();
```

Comment:
Didn't you still want to delete any entries that might be in this map?

Reply:
Ah, I see: Don't test for emptiness but still clear exisitngParms. Sorry, I misunderstood you.